jgclark.NoteHelpers/plugin.json

```diff
@@ -139,6 +139,14 @@
       "description": "Lists the names of notes whose filenames are inconsistent with their titles.",
       "jsFunction": "listInconsistentNames"
     },
+    {
+      "name": "reset title to match filename",
+      "alias": [
+        "filenameToTitle"
+      ],
+      "description": "Resets the current note title to match its filename.",
+      "jsFunction": "filenameToTitle"
+    },
```

Comment:
See code below for implementation details. This adds a new command for setting the filename (excluding extension) as the note title.

<details><summary>Filename</summary>
<img width="254" alt="SCR-20240208-umum" src="https://github.com/NotePlan/plugins/assets/266852/13a6e568-321d-476f-8d91-f6217d8aa8e9">
</details> 

<details><summary>Noteplan command</summary>
<img width="739" alt="SCR-20240208-umiv" src="https://github.com/NotePlan/plugins/assets/266852/bd3c36b4-2d4c-49dd-9df3-fddd8f6cbba9">
</details> 

<details><summary>Confirmation prompt</summary>
<img width="308" alt="SCR-20240208-umrq" src="https://github.com/NotePlan/plugins/assets/266852/a5134869-95a3-4099-990f-82519d90fdce">
</details>